pkgs/development/libraries/gmp/6.x.nix

```diff
@@ -29,7 +29,7 @@ let self = stdenv.mkDerivation rec {
   passthru.static = self.out;
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
-  nativeBuildInputs = [ m4 ];
+  nativeBuildInputs = [ m4 ] ++ lib.optional stdenv.hostPlatform.isWindows autoreconfHook;
```

Comment:
Can you add a comment why this is needed? I.e. is this a structural problem or will this fix itself in a future gmp release?